src/CppUTest/TestRegistry.cpp

```diff
@@ -50,6 +50,7 @@ void TestRegistry::runAllTests(TestResult& result)
     result.testsStarted();
     for (UtestShell *test = tests_; test != NULL; test = test->getNext()) {
```

Comment:
The isRunIgnore is not called everywhere outside the class, so adding it is not useful. In fact, it shouldn't have been added if the code was test-driven....

Also the member varialble is never used and can be moved down in the class hierarchy. I recommend to do that as it isn't needed to have that state in the UtestShell.

I wish we could move the setRunIgnored also out of the UTestShell interface, but I wouldn't know how. The reason is that usually this kind of information is given at object construction time, but the UtestShell is constructed at program start, so that isn't possible. So, I guess we'll need to keep that *one* method in the UtestShell interface.